hr_attendance_management/models/hr_attendance_break.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Stephane Eicher <seicher@compassion.ch>
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields, api
+
+
+class HrAttendanceBreak(models.Model):
+    _name = "hr.attendance.break"
+    _description = "Attendance break"
+
+    ##########################################################################
+    #                                 FIELDS                                 #
+    ##########################################################################
+
+    name = fields.Char(compute='_compute_name')
+    employee_id = fields.Many2one('hr.employee', "Employee", required=True)
+    attendance_day_id = fields.Many2one(
+        'hr.attendance.day', "Attendance day", required=True,
+        ondelete="cascade")
+
+    original_duration = fields.Float(
+        compute='_compute_original_duration', store=True)
+    additional_duration = fields.Float('Duration added by the rules')
+    total_duration = fields.Float(
+        'Break duration', readonly=True, compute='_compute_total_duration',
+        store=True, oldname='logged_duration')
+
+    is_offered = fields.Boolean()
+    system_modified = fields.Boolean(
+        'Modified by the system', compute='_compute_system_modified')
+
+    start = fields.Datetime(compute='_compute_start_stop', store=True)
+    stop = fields.Datetime(compute='_compute_start_stop', store=True)
+
+    previous_attendance = fields.Many2one('hr.attendance')
+    next_attendance = fields.Many2one('hr.attendance')
+
+    ##########################################################################
+    #                             FIELDS METHODS                             #
+    ##########################################################################
+    @api.multi
+    def _compute_name(self):
+        for rd in self:
+            if rd.is_offered:
+                rd.name = 'Free break'
+            elif not rd.start:
+                rd.name = 'Created by the system'
+            else:
+                start = fields.Datetime.from_string(rd.start)
+                start = fields.Datetime.context_timestamp(
+                    rd.employee_id, start)
+                start = fields.Datetime.to_string(start)[-8:]
+                stop = fields.Datetime.from_string(rd.stop)
+                stop = fields.Datetime.context_timestamp(
+                    rd.employee_id, stop)
+                stop = fields.Datetime.to_string(stop)[-8:]
+                rd.name = start + ' - ' + stop
+
+    @api.multi
+    @api.depends('previous_attendance', )
```

Comment:
It's not necessary because at each attendance write, all the attendance_break of the attendance_day are deleted and recreate. I know is not the best method to do... but I don't have the time to change it.